### PR TITLE
4.4 Solo Challenge - completed

### PR DESCRIPTION
**Pull request by rdfm**

### CODE REVIEW

**Comment by rdfm** [comment]
@mechamanny Request for code review. Thank you!

File: ruby/vampires.rb [hunk 1]
```diff
@@ -1,92 +1,89 @@
...
+	user_age = gets.to_i
+
+	puts "What year were you born?"
+	user_year = gets.to_i
+
+	calc_age = 2016 - user_year 
+	if user_age == calc_age
+		age = true
+	else
+		age != true
```
Comment (mechamanny) [inline ruby/vampires.rb]: This is not a valid assignment. You can do `age = false` instead.

File: ruby/vampires.rb [hunk 2]
```diff
@@ -1,92 +1,89 @@
...
+	case user_health
+	when 'Y'
+		health = true
+	when 'N'
+		health = false
+	end
+
+	#Alergies
+	puts "Name an alergy? (List 1 at a time / Type: done - when finished)"
+	while alergy = gets.chomp.upcase
```
Comment (mechamanny) [inline ruby/vampires.rb]: this is not really a good conditional to use for staying in the while loop. Maybe gather the allergy before the `while`, then you can write something like `while allergy != 'DONE' || allergy != 'SUNSHINE'`. Then collect the user input again before the loop exits.

File: ruby/vampires.rb [hunk 3]
```diff
@@ -1,92 +1,89 @@
...
+	end
+
+	# Vampire Result
+	case
+	# If the employee got their age right, and is willing to eat garlic bread or sign up for insurance, the result is “Probably not a vampire.”
+	when age && (garlic_bread || health)
+		puts "RESULT: Probably not a vampire"
+
+	# If the employee got their age wrong, and hates garlic bread or waives insurance, the result is “Probably a vampire.”
+	when (!age && !garlic_bread && health) || (!age && garlic_bread && !health) || !alergy
```
Comment (mechamanny) [inline ruby/vampires.rb]: If the test does not care if `garlic_bread` is true or false, you don't have to include it in your conditional.

Comment (mechamanny) [inline ruby/vampires.rb]: This can be simplified.

**Pull request merged**